src/main/java/test/runModel.java

```diff
@@ -0,0 +1,18 @@
+package test;

```

Comment:
This could be helpful if users want to use the spatial housing market model as a component of their own integrated models, as they may have to run the market model outside of its package.

Reply:
Hmmm, do you mean that we will need to keep the main outside of the package? I'm happy to accept this change in the longer term, when there will be any expectation of any external user. However, to be honest, I don't expect anybody to be using this package at the moment or in the near term... so I don't know...

Reply:
Yes, no worries! I used it here only for test purpose and would be happy to remove it.